jerry-main/main-unix.c

```diff
@@ -45,11 +45,19 @@ static const uint8_t *
 read_file (const char *file_name,
            size_t *out_size_p)
 {
-  FILE *file = fopen (file_name, "r");
-  if (file == NULL)
+  FILE *file;
+  if (strcmp("-", file_name))
```

Comment:
Other parts the `strcmp` is used wiht `!`. Can't we just have the same variant here?